conans/test/release_congruency.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
```

Comment:
Move this to the "test" repo. Ask me if you don't know where to put it.

Reply:
Done, check the PR there: https://github.com/conan-io/test/pull/8